bubblesub/ui/main_window.py

```diff
@@ -208,6 +218,30 @@ def _setup_menu(self) -> None:
             HotkeyContext.Global,
         )
 
+    def _restore_widgets_visibility(self) -> None:
+        for (key, data) in self._api.cfg.opt["gui"]["visibility"].items():
+            widget = self.findChild(QtWidgets.QWidget, key)
+
+            if data[1]:
+                widget = widget.parentWidget()
+
+            widget.setEnabled(data[0])
+            widget.setVisible(data[0])
+
+    def _store_widgets_visibility(self) -> None:
+        def _store_widget(widget_name: str, childs: bool):
+            widget = self.findChild(QtWidgets.QWidget, widget_name)
+
+            if childs:
```

Comment:
children*
And once we go with `console-container` approach it won't be needed anymore